arch/mips/src/pic32mx/pic32mx_lowinit.c

```diff
@@ -155,6 +156,60 @@ static inline void pic32mx_cache(void)
   asm("\tmtc0 %0,$16,0\n" : : "r" (regval));
 }
 
+/****************************************************************************
+ * Name: pic32mx_disable_jtag
+ *
+ * Description:
+ *   Disable the JTAG connection
+ *
+ * Assumptions:
+ *   Interrupts are disabled.
+ *
+ ****************************************************************************/
+
+static inline void pic32mx_disable_jtag(void)
```

Comment:
Should new interfaces be wrapped with ifdef to avoid compilation warning about unused code?

Reply:
I wasn't sure if these should be public functions at all, so I simply made them static for simplicity.  From the datasheet it seems to indicate that you can turn JTAG on and off at runtime(although exactly why you would do that is a mystery to me).